src/operations/device/Resolvers.js

```diff
@@ -4,6 +4,8 @@ import getDeviceHistoryForDashboard from './query.getDeviceHistoryForDashboard.j
 import createDevice from './mutation.createDevice.js';
 import deleteDevices from './mutation.deleteDevices.js';
 import editDevice from './mutation.editDevice.js';
+import favoriteDevices from './mutation.favoriteDevices.js';
+import getFavoriteDevices from './query.getFavoriteDevices.js';
```

Comment:
This function is not used and it seems that this file also doesn't exist :thinking: